database/db_build.sql

```diff
@@ -0,0 +1,30 @@
+BEGIN;
+
+DROP TABLE IF EXISTS users, goals, ratings cascade;
```

Comment:
Are you sure want want to wipe the db when you run this script. I could be wrong here, but this feels unsafe. This could end up staying in source then causing accidental damage down the line.